commands/cartridge.go

```diff
@@ -0,0 +1,48 @@
+package commands
+
+import (
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/tarantool/cartridge-cli/project"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	projectCtx project.ProjectCtx
+	rootCmd    = &cobra.Command{
+		Use:   "cartridge",
+		Short: "Tarantool Cartridge command-line interface",
+	}
+
+	verboseLogLevel bool
+)
+
+func init() {
+	rootCmd.PersistentFlags().BoolVar(&verboseLogLevel, "verbose", false, "Verbose output")
+
+	initLogger()
+}
+
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		log.Fatalf(err.Error())
```

Comment:
I think one day we should do as it was done it 1.x versions - pretty error message with message that it's an error and place where it could be reported.

Reply:
Yes, I sure that we need to handle internal errors and format them. 
But here is handled runtime errors, such as non-existent directory and so on.